app/helpers/summary_document_helper.rb

```diff
@@ -0,0 +1,12 @@
+module SummaryDocumentHelper
+  def link_to_summary_document(appointment)
+    return if appointment.no_show? || appointment.cancelled_by_customer? || appointment.cancelled_by_pension_wise?
```

Comment:
`#cancelled?` encapsulates the last two `cancelled_*` checks so you can shorten this a little. Also can you add a bit of whitespace under the guard condition so it's called out a little clearer?